backend/tests/test_api/test_export_endpoints.py

```
@@ -387,3 +387,109 @@ async def test_pdf_export_returns_503_when_unavailable(self, auth_client_factory
+        assert all("margins" in t and "font" in t for t in templates)
+
+    @pytest.mark.asyncio
+    async def test_formats_endpoint_includes_templates(self, book):
+        client, book_id = book
+        resp = await client.get(f"/api/v1/books/{book_id}/export/formats")
+        assert resp.status_code == 200
+        assert len(resp.json()["templates"]) >= 3
+
+    @pytest.mark.asyncio
+    async def test_templates_endpoint_requires_auth(self, book):
+        client, book_id = book
```

Ruff (0.15.18)

[warning] Unpacked variable `book_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)